.golangci.yml

```diff
@@ -29,12 +30,19 @@ linters-settings:
     # Correct spellings using locale preferences for US or UK.
     # Default is to use a neutral variety of English.
     # Setting locale to US will correct the British spelling of 'colour' to 'color'.
+
+  tagliatelle:
+    # Check the struct tag name case.
+    case:
+      rules:
+        # See https://github.com/ldez/tagliatelle for values and examples:
+        json: goCamel
```

Comment:
Not in theme with the rest of the PR, but I found one structure that was still using snake case in the JSON tags. This will prevent it from happening in the future.